ethereum/sign_test.go

```diff
@@ -0,0 +1,31 @@
+// +build !js
```

Comment:
Hmm.. What's the blocker for testing this in Node.js/Wasm? Does github.com/ethereum/go-ethereum/rpc not work out of the box?

Reply:
It's just that the RPC call happens in a test file and Golang re-routes http requests in test files: https://github.com/golang/go/issues/31559